src/js/components/Footer/index.d.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import { BoxProps } from '../Box/index';
```

Comment:
I believe we have the same in `Header` in the `index.d.ts` can we update that ?

Reply:
Yeah, Header is using `BoxTypes` which includes the `children` prop but to keep Header consistent with the other files I updated it to use `BoxExtendedProps`

Reply:
yeah that's what I meant to add in the `BoxExtendedProps` for Header thank you 😄